engine/src/main/java/org/terasology/rendering/primitives/ChunkMesh.java

```diff
@@ -56,6 +57,7 @@
     /**
      * Possible rendering types.
      */
+    @API
```

Comment:
I don't understand enough about the rendering pipeline to decide whether this should be exposed as `API` or not… the render types seem a bit arbitrary, especially `WATER_AND_ICE`.

Reply:
I'm also wondering what the use-case would be for exposing the render types.